spec/requests/api/v1/users/sign_up_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe 'User acount creation', type: :request do
+  before { post api_v1_user_registration_path, params: user_params, as: :json }
+
+  context 'when attributes are good' do
+    let(:user_email) { Faker::Internet.email }
+    let(:user_params) do
+      { email: user_email, password: Faker::Internet.password }
+    end
+
+    it 'creates an user account' do
+      expect(response).to have_http_status(:created)
+      expect(User.count).to be(1)
```

Comment:
I'm not sure if this is correct here... when you create a new user, you expect to have 'one more', not 'just one' (even when this works in this case, you'll have only one here because of the way tests work)

Reply:
You are correct! It should be one more